Vk final edits after reading pdf (#117)
drvinceknight/amwoss: src/chapters/01/main.tex

If you are a student of a mathematical discipline, a graduate student of
a subject like operational research, a hobbyist who enjoys solving the
travelling salesman problem or even if you get paid to do this stuff:
this book is for you. We will introduce you to the world of open source % TODO
% Is Introduce the correct word here? Does introduce imply they need to not know
% any python or R to start with this book? (Which is not correct). I cannot
% think of a better word, perhaps: "show"?
software that allows you to do all these things freely.

If you are a student learning to write code, a graduate student using

> databases for their research, an enthusiast who programmes applications

Author: ```suggestion
databases for their research, an enthusiast who programs applications
```

        \label{fig:rsd-triangle}
    \end{center}
\end{figure}

> % TODO Do we want something here to introduce the reader to the rest of the
> % book?
> % "Thank you for picking up this book, we hope one if not all of the following
> % chapters proves interesting and useful to you.

Author: ```suggestion
Thank you for picking up this book, we hope one if not all of the following
chapters proves interesting or useful to you.
```
